pkg/util/net/ip/ip.go

```diff
@@ -19,10 +19,13 @@
 package ip
 
 import (
+	"fmt"
 	"io/ioutil"
 	"net"
 
 	netutils "k8s.io/utils/net"
+
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
```

Comment:
This causes our general purpose ip package to depend on a specific virt-launcher API, which seems improper to me.
Generally speaking, this package should not depend on anything internal except what is under "pkg/util".